sample-apps/celery_sample/requirements.txt

```diff
@@ -0,0 +1,10 @@
+asgiref
+certifi
+charset-normalizer
+Django
+gunicorn
+idna
+judoscale-python
+requests
+sqlparse
+urllib3
```

Comment:
I had to also install `celery` and `redis` packages. Should they be in here?

Reply:
Yes, I wil add these in the requirements